index.js

```diff
@@ -0,0 +1,114 @@
+import assert from 'node:assert'
+import ms from 'ms'
+
+export class CancelStuckTransactions {
```

Comment:
Nitpick: class names should be nouns.

E.g.

```suggestion
export class StuckTransactionsCanceller {
```